src/Command.ts

```diff
@@ -250,6 +250,8 @@ export abstract class Command
         {
             const embeds = [ embedify(`Couldn't run the command due to an error${err instanceof Error ? `: ${err.message}` : "."}`, settings.embedColors.error) ];
 
+            console.error(`Error while running the command ${k.yellow(`/${int.commandName}`)}\n`, err);
```

Comment:
idk why we didn't add this sooner, imo it's really useful

Reply:
smh
https://github.com/codedrunks/BrewBot/pull/79/files#diff-dd56303679f34493449b477eeb5f9b1b85bc2c7f35227dee53267ce565601238

Reply:
wow I am schizophrenia

Reply:
mine looks better tho